Unity-Technologies/com.unity.multiplayer.samples.coop
# fix: imps self-destroying on clients when late-joining with multiple additive scenes loaded

### Description (*)

This PR makes sure ClientCharacterVisualization is only enabled on clients when it is spawned. This prevents Update from being called at the wrong time, causing it to self-destruct. This PR also removes an unneeded property and moves the OnDestroy code to OnNetworkDespawn to better mirror its initialization in OnNetworkSpawn.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-2433](https://jira.unity3d.com/browse/MTT-2433)
### Manual testing scenarios

1. Start a game as a host
2. Start the game and move to the second door to load the TransitionArea additive scene
3. As a client, join the game instance
4. See that the imps in the first room are visible
### Questions or comments

### Contribution checklist
